sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -338,6 +338,14 @@ private[hive] class HiveClientImpl(
       // partition columns are part of the schema
       val partCols = h.getPartCols.asScala.map(fromHiveColumn)
       val schema = h.getCols.asScala.map(fromHiveColumn) ++ partCols
+
+      // Skew spec, storage handler, and bucketing info can't be mapped to CatalogTable (yet)
+      val fullyMapped = (
+        h.getSkewedColNames.isEmpty
+          && h.getStorageHandler == null
+          && h.getBucketCols.isEmpty
+      )
```

Comment:
nit: can you use `{ }` here instead? We usually don't use `( )` for things like these